### DISCUSSION TITLE
Copilot tab shortcut not working for markdown files in Visual Studio 2022 Community

### DISCUSSION THREAD

**Discussion by kkirkfield** [community/community#54287] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I've seen plenty of issues reported about the tab shortcut not working in VS Code and there are workarounds for that.

I'm running into a similar issue in Visual Studio 2022 Community that does not appear to have any posted work arounds for.

When editing a markdown file in Visual Studio, pressing tab will insert a tab in the middle of the line. It does not insert the tab at the start of the line, and it does not accept the Copilot suggestion.

**Comment by Akash1134** [comment]
Hi @kkirkfield ,

The first thing we need to ensure is that you have Copilot correctly configured in Visual Studio. Please refer [Configuring GitHub Copilot in your environment](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment?tool=visualstudio)

After ensuring the configuration is correct, if you're still having issues, please share the logs from Visual Studio so that we can take a look:
[Troubleshooting GitHub Copilot in Visual Studio](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio)

Thanks!

**Reply by SerratedSharp** [reply]
I was able to reproduce this with most recent version of VS2022 community Github copilot.  It should be easy to reproduce.  While editing a markdown file, if your cursor is anywhere other than the end of a line, pressing Tab indents the line.  If copilot is providing a suggested completion, pressing tab indents the line rather than accepting the suggestion.  It most other editors the default behavior is for tab to accept the suggestion, and if you want to actually Tab rather than accept the suggestion, then you first hit Esc then Tab.  So other editors have a method that allows you to choose either to either accept suggestion or actually Tab.  Markdown files are being treated the other way around, and there's not really any workaround that I know of to force it to accept the suggestion in leu of Tabbing.

**Reply by GuidoNeele** [reply]
Yes I can confirm this is a problem. Copilot suggestions can't be accepted when you're editing a list in markdown.

**Reply by ayim** [reply]
@GuidoNeele @SerratedSharp we're looking into this.
Tracking work on this feedback item: https://developercommunity.visualstudio.com/t/Tab-does-not-work-for-GitHub-Copilot-in-/10530824

**Reply by howprice** [reply]
I am seeing this with latest Visual Studio 2022 Community. It's very frustrating not to be able to accept the suggested text in the Markdown. Any update on the investigation/solution or a workaround?

**Reply by KarlMaynardDK** [reply]
I had this same issue when using markdown monster as the editor, but not when using the build in vs2022 markdown editor... I changed by simply right clicking the file and asking it to use the "visual studio markdown editor" from the open with menu.
